src/services/signedTransaction.ts

```diff
@@ -1,65 +1,36 @@
 import config from "config";
 import axios from "axios";
 import { Request, Response } from "express";
+import { calculateTxId } from "../utils";
 
 const submissionEndpoint :string = config.get("server.txSubmissionEndpoint");
 
 // THIS IS FOR CARDANO-WALLET, CBOR IS FOR CARDANO-SUBMIT-API (1.27.0).
 const contentTypeHeaders = {
-  "Content-Type": `application/${process.env.TX_SUBMIT_CONTENT_TYPE || "octet-stream"}`
+  "Content-Type": "application/json"
```

Comment:
@pedromtcosta , I have forgot to clarify, this change must be done in a backward-compatible way as other services are using our backend codebase as well when working with cardano-wallet or cardano-submit-api for sending. Plz add a new env variable, denoting if the submit target is yoroi-queue, and use new logic only in case when it's so. The existing logic must be preserved the way it worked before.

Reply:
Done! See latest pushes